AuxTel/AuxTel-Troubleshooting/General-Troubleshooting/Images-out-of-focus.rst

```diff
@@ -0,0 +1,128 @@
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Carlos Morales*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Ioana Sotuela, Karla Aubel*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Image-out-of-focus-Procedure:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+Image out of focus
+#########################
+
+.. note::
+    This is a procedure template file that is associated with a template directory structure. This note should be deleted when the section is properly populated.
+
+.. _Image-out-of-focus-Procedure-Overview:
+
+Overview
+========
+
+.. In one or two sentences, explain when this Image out of focus procedure needs to be used. Describe the symptoms that the user sees to use this procedure. 
+
+This procedure should be used when images appear out of focus due to a WEP failure or other related issues. Users will notice images ranging from slightly out of focus to large donut appearances when hexapod offsets are not correctly applied.
+
+.. _Image-out-of-focus-Procedure-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+.. This section should provide simple overview of known or suspected causes for the error.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Post screenshots of the error state or relevant tracebacks.
+
+- The focus script `latiss_wep_align` might fail, leaving the system out of focus.
+- Out-of-focus images can appear as large donuts when hexapod offsets are left in intra or extra hexapod positions due to script failure.
```

Comment:
1. intra or extra hexapod positions ---> intra- or extra-hexapod positions
2. I think the accurate way of saying this intra- or extra- focus positions (but I maybe wrong)